src/jsMain/kotlin/io/beatmaps/user/main.kt

```diff
@@ -135,137 +148,226 @@ class ProfilePage : RComponent<ProfilePageProps, ProfilePageState>() {
     private fun setFollowStatus(followed: Boolean) {
         Axios.post<UserFollowRequest>("${Config.apibase}/users/follow", UserFollowRequest(state.userDetail?.id ?: 0, followed), generateConfig<UserFollowRequest, String>()).then({
             setState {
-                following = followed
+                followData = UserFollowData(
+                    (followData?.followers ?: 0).let {
+                        if (followed) it + 1 else it - 1
+                    },
+                    followData?.follows ?: 0,
```

Comment:
Probably shouldn't have the default value here. Causes the info line to appear when following/unfollowing users.

Reply:
You're right, that wasn't meant to be there.